acorn/src/expression.js

```diff
@@ -505,11 +505,11 @@ pp.parseExprAtom = function(refDestructuringErrors, forInit, forNew) {
     }
 
   default:
-    return this.parseExprAtomDefault()
+    return this.parseExprAtomDefault(forInit)
```

Comment:
These changes seem unrelated to the rest of the patch, and `parseExprAtomDefault` does not, at this point, take any arguments.